src/main/java/org/spongepowered/api/item/inventory/InventoryProperties.java

```diff
@@ -78,22 +85,22 @@
             DummyObjectProvider.createFor(Property.class, "SLOT_POSITION");
 
     /**
-     * A property which represents a "side" for a particular slot,
-     * for use in querying "sided inventories".
+     * A property which represents a "side" for a particular slot, for use in querying "sided inventories".
      */
     public static final Property<Direction> SLOT_SIDE =
             DummyObjectProvider.createFor(Property.class, "SLOT_SIDE");
 
     /**
      * Represents the {@link UUID} of a inventory.
      */
+    // TODO not present in impl for query atm.
     public static final Property<UUID> UNIQUE_ID =
             DummyObjectProvider.createFor(Property.class, "UNIQUE_ID");
 
     /**
-     * Represents the title of a inventory, is
-     * viewable by players looking at the inventory.
+     * Represents the title of a inventory, is viewable by players looking at the inventory.
      */
+    // TODO maybe not present in impl? most inventories dont have a title. also title only shows in Containers
     public static final Property<Text> TITLE =
```

Comment:
Getting a properties' value returns an Optional, yes? If so, this is fine to keep. Maybe document that all of these properties are not guaranteed to exist on all Inventories.